manifests/j/JackDevey/Lux/1.1.1/JackDevey.Lux.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: JackDevey.Lux
+PackageVersion: 1.1.1
+PackageLocale: en-US
+Publisher: Jack Devey
```

```suggestion
Publisher: BanDev
```
Looks like the developer hasn't updated their installer to change the publisher to be `Jack Devey` yet which means that WinGet won't recognize the application so we need to keep this as `BanDev` for now until it has been changed.